README.md

```diff
@@ -1,39 +1,34 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Getting started
-
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+### How to use this app
+1. Obtain a key and code from Trello API, set this in the .env file
+2. Get your board ID from tello and set this in the .env file
+3. Obtain from trello your list id's and set these in the *_list_id variables 
 
-### On macOS and Linux
 ```bash
-$ source setup.sh
+trello_key=
+trello_token=
+trello_boardid=
+todo_list_id=
+doing_list_id=
+done_list_id =
 ```
-### On Windows (Using Git Bash)
+### Building Docker Image
 ```bash
-$ source setup.sh --windows
+docker build --target development --tag todoapp:dev .
+docker build --target production --tag todoapp:prod .
 ```
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
+## Running Docker Image as Production
+To run the ToDo app as production, run the following
 ```bash
-$ flask run
+docker run -p 80:5000 --env-file .env -d todoapp:prod
 ```
+Go to http://localhost:80 and you should see the ToDo app now running
 
-You should see output similar to the following:
+## Running Docker Image as Development
+To run the ToDo app as production, run the following
 ```bash
- * Serving Flask app "app" (lazy loading)
- * Environment: development
- * Debug mode: on
- * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
- * Restarting with fsevents reloader
- * Debugger is active!
- * Debugger PIN: 226-556-590
+docker run -p 80:5000 --env-file .env --mount type=bind,source=$(pwd),target=/usr/src/app todoapp:dev
```

Comment:
I think the bind mount here is incorrect, possibly a name change or a copy-paste error, but as it stands the target directory doesn't match the code directory so the hot reloading doesn't quite work